cucumber/api/features/step_definitions/policy_load_steps.rb

```diff
@@ -0,0 +1,69 @@
+require 'English'
+
+When(/^I load a large policy with POST$/) do
+  # Generate a large policy with 1000 variables
+  policy_body = "- &my-variables\n" + [*1..1000].map do |ndx|
+    %Q(
+      - !variable
+        id: obj_Safe_00c_0_#{ndx}/password
+        annotations:
+          cyberark-vault: 'true'
+          cyberark-vault/accounts: AutomationVault/Safe_00c/obj_Safe_00c_0_10
+    )
```

Comment:
Where does this name come from: `Safe_00c/obj_Safe_00c_0_10`?

Reply:
This comes from the synchronizer test policy. It's the name of a vault and safe in Enterprise Password Vault (EPV). It doesn't have a significant meaning here, other than is representative of the failing policy document.